test/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
 
-project(GreeterTests LANGUAGES CXX)
+project(
+  GreeterTests
+  VERSION 1.1
```

Comment:
I don't think the tests need a version.
- It adds more code to maintain
- The version is already defined in the main project

Reply:
it fails, if in my case, wenn the 1.0 version was installed, but 1.1 not!